test/langtools/jdk/javadoc/lib/javadoc/tester/JavadocTester.java

```diff
@@ -543,6 +557,26 @@ private void checkOutput(String path, String fileString, boolean expectedFound,
         }
     }
 
+    private static boolean checkIfPrefixes(String... strings) {
+        String[] copy = Arrays.copyOf(strings, strings.length);
+        Arrays.sort(copy);
+        for (int i = 0; i < copy.length - 1; i++) {
+            if (isFirstPrefixOfSecond(copy[i], copy[i + 1]))
+                return true;
+        }
+        return false;
+    }
+
+    static boolean isFirstPrefixOfSecond(String a, String b) {
+        int i = 0;
+        int j = 0;
+        while (i < a.length() && j < b.length() && a.charAt(i) == b.charAt(j)) {
+            i++;
+            j++;
+        }
+        return i == a.length();
+    }
+
```

Comment:
Why not `String.startsWith` ?

Reply:
Good catch! I forgot that I need a qualitative answer not the quantitative (e.g. length of the common prefix).